query/ad_third_party_application_not_allowed.sql

```diff
@@ -0,0 +1,17 @@
+-- Ensure third party integrated applications are not allowed
+select
+  -- Required columns
+  tenant_id as resource,
+  case
+    when default_user_role_permissions ->> 'allowedToCreateApps' = 'false' then 'ok'
```

Comment:
Could we use `(default_user_role_permissions -> 'allowedToCreateApps')::bool` rather than testing against the string false in these cases?

Using a string `'false'` rather than a bool `false` feels more error prone and like it's fighting with the type system to me?

Reply:
Yes Sure, fixed. I have used like this --> 
`
 when not (default_user_role_permissions -> 'allowedToCreateApps')::bool then 'ok'
`